javascript/funWithFunctions.md

```diff
@@ -0,0 +1,24 @@
+## Functional Expressions in D&D ##
```

Comment:
This once appeared as a blog post: http://robots.thoughtbot.com/for-all-you-budding-programmers